doc/Known_issues/Troubleshooting.md

```diff
@@ -51,7 +51,11 @@ If you encounter any kernel panics or service crashes, debugging dump files may
 * Kernel crashdumps: `$Env:SystemRoot\MEMORY.DMP` and `$Env:SystemRoot\Minidump\*dmp`
 * Usermode crashdumps: `$Env:LocalAppData\CrashDumps`
 
-## 8. Total cleanup
+## 8. PDB files
+
+Debugging symbols for Contrail Components can be found in PDB files in Artifacts folder, next to MSI files. Artifacts folder is unpacked from docker containers on test beds and can be found directly on C: drive.
```

Comment:
[pedantic] Could you please split this line?

Also, we should describe in a sentence or two what tools we can use to analyze these I think